cf/data/collapse/collapse.py

```diff
@@ -113,7 +156,7 @@ def max_abs(
         a,
         axis=None,
         keepdims=False,
-        mtol=None,
+        mtol=1,
```

Comment:
I see - `mtol` must be number, so `mtol=None` is really a bug. It wasn't causing problems in the `Collapse` methods because of the defaults all being `1` in `Data`, which sets up and calls the `Collapse` methods. Fixed in: dd92b55d343f7f341f7fdbf78bf12f1e988c3929

Reply:
OK fair, thanks for clarifying. The fix looks good. This (`None` meaning a bug) is just the case for `mtol` for collapse methods, right? Else there are various `atol=None` (`git grep "atol=None" | wc -l` gives `35`) that might need converting, too.

Reply:
Yes - it's just for `mtol`. `atol=None` means "use cf.atol()" - whether or not that's a good API is not relevant here!

Reply:
OK makes sense. Marking as resolved.